### ISSUE TITLE
How and when to report issue on github [meta]

### ISSUE THREAD

**Issue by dz0ny** [mopidy/mopidy#830]
From disscusion on irc:

https://github.com/ansible/ansible/blob/devel/CONTRIBUTING.md#reporting-a-bug
https://github.com/drush-ops/drush/issues/new

My proposal is to:
a) get users to discuss problems on discuss.mopidy.com
b) if problem is of technical nature and sw fault(demonstrated), encourage user to file proper report using template
c) individual devs might want to clarify in their profile pages, what is appropriate reason, for someone to contact them on their personal/public mail or any other means of communication

TLDR: Github issues section is not support forum. We should have template for issue reporting

**Comment by jodal** [comment]
Another one for inspiration: https://github.com/jeremyevans/sequel/blob/master/CONTRIBUTING

**Comment by adamcik** [comment]
Perhaps something like:

Hi! Nice to see you here!

If you'd like to ask a question
======================

If you get stuck, you can get help at the Mopidy discussion forum. We also hang around at IRC on the #mopidy channel at irc.freenode.net. The IRC channel has public searchable logs.

If in doubt try one of these first as the issue tracker is for tracking bugs and code changes, not support.

If you'd like to contribute code
=======================

Please see [this web page](https://docs.mopidy.com/en/develop/contributing/) for information about the contribution process.

If you'd like to file a bug
==================

If you stumble into a bug or have a feature request, please create an issue in the issue tracker. If you’re unsure if it’s a bug or not, ask for help in the forum or on IRC first. https://docs.mopidy.com/en/develop/troubleshooting/ also has some useful pointers for debugging.

Thanks!

**Comment by jodal** [comment]
...based upon Ansible's `CONTRIBUTING.md`, to give credit where credit is due.

**Comment by adamcik** [comment]
Mhm, with the rest copied out of our own docs.

**Comment by dz0ny** [comment]
*Mopidy discussion forum* should be a link :)

**Comment by adamcik** [comment]
Linkified some things and updated the suggestion a bit, I know @jodal was already working on something so I trust he'll steal whatever good parts are in mine and end up with something awesome :-)
